backend/cellguide/pipeline/computational_marker_genes/computational_markers.py

```diff
@@ -42,6 +43,12 @@
 
 MARKER_SCORE_THRESHOLD = 0.5
 
+# This file contains blacklisted marker genes that match the following criteria:
+# Ensembl 104 gene IDs that are annotated with the GO terms: ['GO:0005840', 'GO:0005739']
+# These GO terms correspond to the cellular components "ribosome" and "mitochondrion".
+file_dir = os.path.dirname(os.path.realpath(__file__))
+MARKER_GENE_BLACKLIST_FILENAME = os.path.join(file_dir, "marker_gene_blacklist.txt")
```

Comment:
i assume this file was written using a script. do you have the script you used to generate this? might make sense to put it in the `scripts/` directory so it can be used to regenerate the marker gene blacklist if we decide we want to blacklist more types of marker genes

Reply:
It wasn't written by a script as I was not aware of an easy way to get these genes through an API, though I'm sure one does exist.

Reply:
gotcha - how did you generate the file then?

Reply:
Biomart downloads the results as `mart_export.txt` files, one per organism, which contain newline-separated gene IDs and a header line. I skipped the first line for human, read the entire file, split by newline, unique'd the resulting list, did the same thing for mouse, concatenated the lists, joined by commas and dumped to the file.

Reply:
Not sure if it merits extra tooling. If we ever need to do it again, then it would make more sense to make a script for it the second time around.

Reply:
thanks for the explanation! agreed, i don't think it merits extra tooling. but it's useful to have the explanation on the PR somewhere or ideally as a code comment